src/VisualLinter/Linting/LinterMessage.cs

```diff
@@ -0,0 +1,68 @@
+﻿using jwldnr.VisualLinter.Enums;
+
+namespace jwldnr.VisualLinter.Linting
+{
+    public class LinterMessage
+    {
+        public string Source { get; set; }
+        public int Line { get; set; }
+        public int Column { get; set; }
+        public int? EndLine { get; set; }
+        public int? EndColumn { get; set; }
+        public bool IsFatal { get; set; }
+        public string Message { get; set; }
+        public string RuleId { get; set; }
+        public RuleSeverity Severity { get; set; }
+
+        public MessageRange Range { get; set; }
+        public LinterType LinterType { get; set; }
+
+        public LinterMessage(EslintMessage eslintMessage)
+        {
+            this.Source = eslintMessage.Source;
```

Comment:
Try not to use `this.` where not needed. ReSharper should help with this. If you don't already have it installed I strongly recommend it! 👍